src/main/resources/org/jenkinsci/tracey/RabbitMQTrigger/config.jelly

```diff
@@ -10,6 +10,15 @@
         <f:select/>
     </f:entry>
 
+    <f:entry title="Exchange Type" field="exchangeType">
```

Comment:
As far as I can see we already have title="RabbitMQ exchange name" so let's call this one RabbitMQ exchange type instead